CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   Fix GitHub workflow for publishing a new release
 
+
```

Comment:
unnecessary change?

Reply:
Unnecessary, yes. But I do not want to write twice the same thing in the changelog to make the newly instated CI check pass…

Reply:
ok, makes sense.